app/middlewares/validations.js

```diff
@@ -0,0 +1,16 @@
+const { checkSchema, validationResult } = require('express-validator');
+
+const verifySchema = schema => checkSchema(schema);
+
+const validateResult = (req, res, next) => {
+  const err = validationResult(req);
+  if (err.isEmpty()) {
+    next();
+  } else {
+    res.status(409).send(err.mapped());
```

Comment:
Use `next()` here. And at this point, look how to create a custom error for this type of exception. Also `409` is not the correct code here, use this for reference [HTTP CODES](https://www.restapitutorial.com/httpstatuscodes.html)